src/lib/services/map/markerManager.ts

```
@@ -39,32 +41,32 @@ export class MarkerManager {
+            await this.provider.preloadMarkerLibrary();
         } catch (error: unknown) {
             console.error('Failed to pre-load marker library:', error);
         }
     }
 
     public addMarker(
         id: MarkerId,
-        position: google.maps.LatLngLiteral,
+        position: LatLngLiteral,
         options: MarkerOptions,
     ): Marker | null {
```

ESLint

[error] Replace `⏎········id:·MarkerId,⏎········position:·LatLngLiteral,⏎········options:·MarkerOptions,⏎····` with `id:·MarkerId,·position:·LatLngLiteral,·options:·MarkerOptions`

(prettier/prettier)